cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/store/AppMetadataStoreTest.java

```diff
@@ -87,8 +87,10 @@ public void testOldRunRecordFormat() throws Exception {
     txnl.execute(new TransactionExecutor.Subroutine() {
       @Override
       public void apply() throws Exception {
-        metadataStoreDataset.recordProgramStartOldFormat(program, runId.getId(),
-                                                         RunIds.getTime(runId, TimeUnit.SECONDS), null, null, null);
+        metadataStoreDataset.recordProgramStart(program, runId.getId(),
```

Comment:
Yes, but it uses the same `recordProgramRunning` method. The `recordProgramRunning` method expects there to be a record with `ProgramRunStatus.STARTING`. 

The only difference in this oldFormat method is that the key does not include an application version.